zerver/tests/test_signup.py

```diff
@@ -891,27 +891,6 @@ def test_require_realm_admin(self) -> None:
         self.assertTrue(find_key_by_email(email2))
         self.check_sent_emails([email, email2])
 
-    def test_successful_invite_user_with_notifications_stream(self) -> None:
```

Comment:
We don't need this test anymore since we are treating notifications stream pretty much like any other stream while inviting a user.

Reply:
Sounds good!